jsonargparse_tests/test_parameter_resolvers.py

```diff
@@ -899,6 +902,14 @@ def test_get_params_some_ignored():
         assert_params(get_params(func_given_kwargs), ["p", "p1"], help=False)
 
 
+# test partial method
```

Comment:
Why this comment? It is redundant, since it is the same as the function name.

```suggestion
```

Reply:
sure, removed the comment